### ISSUE TITLE
[Programmatic Access – Make Code - Language] Ensures elements with an ARIA role that require child roles contain them (#availablelocales)

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3663]
**User Experience**:
Screen reader user will get confuse with the control type of the control if require aria child role is not defined for child control appearing under the parent control.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Language " option and select it.
7. Navigate through all the controls appearing under "Select Language" dialog.
8. Run accessibility insights for web tool and select fast pass.
9. Verify the issue.

**Actual Result:**
Aria child role is not defined for the child control which appears under the parent control.

**Expected Result:**
Aria child role should be defined for the child control which appears under the parent control.

**Issue:** Ensures elements with an ARIA role that require child roles contain them (aria-required-children)

**Target application**: Microsoft MakeCode for micro:bit 

**Element path:** #availablelocales

**Snippet:** <div id="availablelocales" class="ui cards centered" role="listbox">

**How to fix:** 
Fix any of the following:
  Required ARIA child role not present: option

This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.3.1; #Win10-Edge(Chromium)

![WCAG1 3 1_Language_Ensures elements with an ARIA role that require child roles contain them](https://user-images.githubusercontent.com/72871089/98902553-f2825e80-24db-11eb-8d0d-aaab6f4c18fa.jpg)
